docs/index.md

```diff
@@ -4,7 +4,7 @@ TorchServe is a performant, flexible and easy to use tool for serving PyTorch mo
 
 
 ## ⚡ Why TorchServe
-* [Model Management API](https://github.com/pytorch/serve/blob/master/docs/management_api.md): multi model management with optimized worker to model allocation
+* [Model Management API](https://github.com/pytorch/serve/blob/master/docs/management_api.md#management-api): multi model management with optimized worker to model allocation
```

Comment:
It seems like the first 3 links are broken in the same way, this is quite strange

<img width="835" alt="Screenshot 2023-05-09 at 5 47 48 PM" src="https://github.com/pytorch/serve/assets/3282513/15e9715d-a6c0-4809-a97e-48e2fe15793e">

Could you share a screenshot of how you tested this? did you build the docs locally? We don't unfortunately have a preview in CI, our docs get built on master directly (something we should fix eventually)

Reply:
Was not sure how to build the docs locally. Let me know if there is a way to do that. I tested the link directly from the index.md (see attached screenshots) and from the FB CMS.

And yes, I noticed that the other links had the same issue. Plan to update them all. Not sure why it happens like this because the links themselves seem sound and if you copy and paste them they work but clicking removes the .md at the end. 

It seems the ones that work (that go to a .md) have an anchor link on them which is why I tried this.

<img width="2435" alt="Screenshot 2023-05-09 at 9 26 13 PM" src="https://github.com/pytorch/serve/assets/127536312/664f9485-4116-4143-8208-61315294f7f3">
![Screenshot 2023-05-09 at 9 26 28 PM](https://github.com/pytorch/serve/assets/127536312/b071f7e8-f293-4720-9933-f0604773cfd6)

Reply:
Ok let's just fix all 3 links at once

To build the docs locally follow these instructions here https://github.com/pytorch/serve/blob/master/.github/workflows/doc-automation.yml#L18-L22

Reply:
Thanks Mark for the information. I had just found that info as well and tested locally. It seems to work. 
Will fix the rest of the links and submit another PR.